mobile/Screen/ProfileScreen.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import {
+  StyleSheet, Text, TextInput, View, Button,
+} from 'react-native';
+import { Picker } from '@react-native-picker/picker';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+    padding: 25,
+    // alignItems: "center",
+    // justifyContent: "center",
+  },
+});
+
+function ProfileScreen() {
+  const [name, onChangeName] = useState('');
+  const [email, onChangeEmail] = useState('');
+  const [platform, setPlatform] = useState('');
+  const [tag, onChangeTag] = useState('');
+  const [profileDetails, setProfileDetails] = useState({}); // on hit clear, store fields in object
+
+  const handleClear = () => {
+    setProfileDetails({
+      name,
+      email,
+      platform,
+      tag,
+    });
```

Comment:
Great way of using an object to simplify saving details! This will be good when editing profile details becomes a feature